plugin/trace/README.md

```diff
@@ -9,6 +9,8 @@
 With *trace* you enable OpenTracing of how a request flows through CoreDNS. Enable the *debug*
 plugin to get logs from the trace plugin.
 
+**Consider using [opentelemetry plugin](../opentelemetry) for tracing of DNS requests, because [OpenTracing project as such is deprecated](https://github.com/opentracing/specification/issues/163).**
```

Comment:
I'm thinking we should deprecate _trace_ now, and remove it in the next minor release.

Reply:
ok, what should be done to deprecate the plugin?

Reply:
If others agree we should deprecate now, we just mention that the plugin is deprecated prominently in the plugin README. And in next release, we'd mention the plugin has been deprecated in the release notes.

Later in a future minor release, we'd remove the plugin.  If I recall, our policy is to wait at lease a couple of releases before removing a plugin/disabling a feature after deprecation.

Reply:
I have added a deprecation notice into trace plugin README 👍